x-pack/plugins/stack_alerts/server/rule_types/es_query/types.ts

```diff
@@ -24,5 +25,6 @@ export type ExecutorOptions<P extends RuleTypeParams> = RuleExecutorOptions<
   EsQueryRuleState,
   {},
   ActionContext,
-  typeof ActionGroupId
+  typeof ActionGroupId,
+  RuleAlertData
```

Comment:
this should be updated to be `StackEsqueryAlert`

Reply:
done